recipe/0326-annotating-image-layer/manifest.json

```diff
@@ -0,0 +1,98 @@
+                "type": "Annotation",
+                "motivation": "painting",
+                "body": {
+                  "type": "Choice",
+                  "items": [
+                    {
+                      "id": "https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-natural/full/max/0/default.jpg",
+                      "type": "Image",
+                      "label": {
+                        "en": [
+                          "Natural Light"
+                        ]
+                      },
+                      "format": "image/jpeg",
+                      "height": 1271,
+                      "width": 2000,
+                      "service": [
+                        {
+                          "id": "https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-natural",
+                          "type": "ImageService3",
+                          "profile": "level1"
+                        }
+                      ]
+                    },
+                    {
+                      "id": "https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-xray/full/max/0/default.jpg",
+                      "type": "Image",
+                      "label": {
+                        "en": [
+                          "X-Ray"
+                        ]
+                      },
+                      "format": "image/jpeg",
+                      "height": 1271,
+                      "width": 2000,
+                      "service": [
+                        {
+                          "id": "https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-xray",
+                          "type": "ImageService3",
+                          "profile": "level1"
+                        }
+                      ],
+                      "annotations": [
+                        {
+                          "id": "https://preview.iiif.io/cookbook/0326-annotating-image-layer/recipe/0326-annotating-image-layer/page/p2/1",
+                          "type": "AnnotationPage",
+                          "items": [
+                            {
+                              "id": "https://preview.iiif.io/cookbook/0326-annotating-image-layer/recipe/0326-annotating-image-layer/annotation/p0002-tag",
+                              "type": "Annotation",
+                              "motivation": "tagging",
+                              "body": {
+                                "type": "TextualBody",
+                                "value": "A group of skulls.",
+                                "language": "en",
+                                "format": "text/plain"
+                              },
+                              "target": {
+                                "source":"https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-xray/full/max/0/default.jpg#xywh=810,900,260,370",
```

Comment:
Cookbook editors suggest changing this to:


https://iiif.io/api/image/3.0/example/reference/421e65be2ce95439b3ad6ef1f2ab87a9-dee-xray/full/2000,1271/0/default.jpg

Reply:
Is this supposed to be without the fragment selector (xywh=810,900,260,370)?

Reply:
No sorry I didn't copy across the fragment selector but that should be in addition to the URI I put above.